### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.keyvault.v2016_10_01.models.key_vault_error.KeyVaultError
  - azure.keyvault.v2016_10_01.models.key_vault_error.KeyVaultErrorException
  fullName: azure.keyvault.v2016_10_01.models.key_vault_error
  langs:
  - python
  module: azure.keyvault.v2016_10_01.models.key_vault_error
  name: key_vault_error
  type: module
  uid: azure.keyvault.v2016_10_01.models.key_vault_error
references:
- fullName: azure.keyvault.v2016_10_01.models.key_vault_error.KeyVaultError
  isExternal: false
  name: KeyVaultError
  parent: azure.keyvault.v2016_10_01.models.key_vault_error
  uid: azure.keyvault.v2016_10_01.models.key_vault_error.KeyVaultError
- fullName: azure.keyvault.v2016_10_01.models.key_vault_error.KeyVaultErrorException
  isExternal: false
  name: KeyVaultErrorException
  parent: azure.keyvault.v2016_10_01.models.key_vault_error
  uid: azure.keyvault.v2016_10_01.models.key_vault_error.KeyVaultErrorException
